frontend/integration-tests/tests/developer-catalog.scenario.ts

```diff
@@ -92,4 +94,39 @@ describe('Catalog', () => {
     const numCatalogTiles = await catalogView.pageHeadingNumberOfItems();
     expect(srvClassFilterCount).toEqual(numCatalogTiles);
   });
+
+  it('creates a service instance and binding', async() => {
+    expect(catalogPageView.catalogTiles.isPresent()).toBe(true);
+
+    await catalogPageView.clickFilterCheckbox('Service Class');
+    await catalogPageView.filterByKeyword('Apache HTTP Server');
+    expect(catalogPageView.catalogTileFor('Apache HTTP Server').isDisplayed()).toBe(true);
+
+    await catalogPageView.catalogTileFor('Apache HTTP Server').click();
+    await catalogView.catalogDetailsLoaded();
+
+    expect(catalogView.createServiceInstanceButton.isDisplayed()).toBe(true);
+    await catalogView.createServiceInstanceButton.click();
+    await catalogView.createServiceInstanceFormIsLoaded();
+
+    await $('#dropdown-selectbox').click();
+    await $$('.dropdown-menu').first().$(`#${testName}-Project-link`).click();
+    await srvCatalogView.createButton.click();
+    await crudView.isLoaded();
+
+    expect(crudView.resourceTitle.getText()).toEqual('httpd-example');
+
+    await catalogView.createServiceBindingButton.click();
```

Comment:
The problem with using this service class for tests is that it's not supposed to be bindable. When we fix the bug to hide the "Create Binding" button for non-bindable service classes, these tests will fail. It would be better to use something like MongoDB that's bindable.